### DISCUSSION TITLE
Documentation regarding the usage of FPM versions of the image

### DISCUSSION THREAD

**Discussion by samip5** [docker-library/wordpress#713] [General]
I'm a little confused as to how one should configure nginx with the FPM version of the image?

**Comment by wglambert** [comment]
Using FPM is a more involved process than can be concisely explained in the image's documentation, and it's specific to the particular web server you'd be using.

Here's an example of using `wordpress:fpm` with `nginx` 
https://github.com/mjstealey/wordpress-nginx-docker/blob/master/docker-compose.yml
https://github.com/mjstealey/wordpress-nginx-docker

If you need further help you could also try asking over at the [Docker Community Forums](https://forums.docker.com/), [Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker). Since these repos aren't really a user-help forum
